### PR TITLE
Landlord_Ruby: Wes

### PR DESCRIPTION
**Pull request by wesareed**
comfort_level: 2.3
completeness: 2

Got the general idea of the MVP fairly easily, although I would like to style it more like a table.

### CODE REVIEW

**Comment by wesareed** [comment]
Still having trouble getting case 3 to work.

File: models/apartment.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Apartment
+  attr_accessor :id, :address, :monthly_rent, :square_feet
+
+  def initialize (id, address, monthly_rent, square_feet)
+    @id = id
+    @address = address
+    @monthly_rent = monthly_rent
+    @square_feet = square_feet
+  end
+end
```
Comment (tylercrosse) [inline models/apartment.rb:10]: 👍

File: models/tenant.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Tenant
+  attr_accessor :id, :name, :age, :apartment_id
+
+  def initialize (id, name, age, apartment_id)
+    @id = id
+    @name = name
+    @age = age
+    @apartment_id = apartment_id
+  end  
+end
```
Comment (tylercrosse) [inline models/tenant.rb:10]: 👍

File: app.rb [hunk 3]
```diff
@@ -1,20 +1,136 @@
...
+    # tenants_order_by_age = tenants.sort_by do |tenant|
+    #   tenant[:age]
+    # end
+    # puts tenants_order_by_age
+
   # Print the names of all the tenants alphabetically
+    # tenants_alpha = tenants.sort_by do |tenant|
+    #   tenant[:name].downcase
+    # end
+    # puts tenants_alpha
```
Comment (tylercrosse) [inline app.rb:71]: 👍

File: app.rb [hunk 4]
```diff
@@ -1,20 +1,136 @@
...
+      puts tenant.name
+      puts "Address:"
+      tenant_address = ruby_apartments.select do |apartment|
+        apartment.id == tenant.apartment_id
+      end
+      tenant_address.each do |tenant|
+        puts tenant.address
+        buffer
+    end
+  end
```
Comment (tylercrosse) [inline app.rb:134]: What you have for 4 is great, but it's not consistent with what you said in menu for 4/the prompt. You could do another `gets.chomp` here, store it in a variable, and then use that to select from tenants.

**Comment by tylercrosse** [comment]
Really nice work! Your CLI app is awesome and I can tell you are getting more comfortable with enumerators.

**Pull request closed without merging**